Example/Tests/Tests.swift

```diff
@@ -58,6 +58,7 @@ class Tests: XCTestCase {
 		XCTAssertEqual(slider.trackShadowDepth, 0)
 		XCTAssertEqual(slider.trackMinAngle, 0)
 		XCTAssertEqual(slider.trackMaxAngle, 360)
+        XCTAssertEqual(slider.areTrackCapsRound, false)
```

Comment:
Seems it's happens again.

Reply:
But in my git client and xCode it looks ok. Very strange.

![Screen Shot 2021-01-29 at 9 41 52 AM](https://user-images.githubusercontent.com/24318356/106246384-b4612500-6216-11eb-9c6b-bec31d9882ba.png)

Reply:
It's spaces vs tabs... I'll help. I'll push a formatting only fix later today and merge it. It's minor anyhow. Good job! 💪

Reply:
It may sound strange, but I copied the tabs that were in the file and pasted them in front of my line of code, and it still happened.

Reply:
It's probably your IDE converting tabs to spaces. That would be my guess 🙂

Reply:
Okay, thanks